permits/forms.py

```diff
@@ -692,11 +692,20 @@ def __init__(self, *args, **kwargs):
 
         super().__init__(*args, **kwargs)
 
-        self.fields["geom"].widget.attrs["options"] = self.get_widget_options(
-            self.permit_request
-        )
+        if not services.geotime_needs_info(self.permit_request, "date"):
+            self.fields.pop("starts_at")
+            self.fields.pop("ends_at")
+
+        if not services.geotime_needs_info(self.permit_request, "geometry"):
+            self.fields.pop("geom")
+        else:
+            self.fields["geom"].widget.attrs["options"] = self.get_widget_options(
+                self.permit_request
+            )
```

Comment:
Tu pourrais ajouter ici l’instruction d’en-dessous: `self.fields["geom"].widget.attrs["options"]["edit_geom"] = not disable_fields`

Reply:
done